src/write_resume_data.cpp

```diff
@@ -141,24 +141,16 @@ namespace {
 				{
 					auto const& verified = atp.verified_leaf_hashes[f];
 					if (!verified.empty())
-					{
-						auto& ret_verified = ret_dict["verified"].string();
-						ret_verified.reserve(verified.size());
-						for (auto const bit : verified)
-							ret_verified.push_back(bit ? '1' : '0');
-					}
+						ret_dict["verified2"] = std::string(verified.data()
```

Comment:
```suggestion
						ret_dict["verified"] = std::string(verified.data()
```

Reply:
this typo exposed a hole in the tests. fixing the tests exposed an issue in this patch where the bitfield sizes are now rounded up to be divisible by 8, which causes failures in the round-trip tests.